SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Vulnerability Disclosure and Response Process
+
+TiDB is a fast-growing open source database. To ensure its security, a security vulnerability disclosure and response process is adopted.
+
+The primary goal of this process is to reduce the total exposure time of users to publicly known vulnerabilities. To quickly fix vulnerabilities of TiDB products, the security team is responsible for the entire vulnerability management process, including internal communication and external disclosure.
+
+If you find a vulnerability or encounter a security incident involving vulnerabilities of TiDB products, please report it as soon as possible to the TiDB security team (security@tidb.io).
```

Comment:
What is the difference between security@tidb.io and tikv-security@lists.cncf.io?

Reply:
Thanks a lot, this is a good question. It seems that tikv and tidb use different Security Policies and vulnerability reporting emails. I will close this PR and re-propose a new PR to migrate tikv Security Policy to this repo.

Reply:
I think we should use the same email as tikv, since it is a project in the tikv org.

Reply:
It is not just choosing an email, but also determining the security policy. If we use the email address security@tidb.io, the security vulnerability of pd will be responded by the tidb security team. If we use the email address tikv-security@lists.cncf.io, which is a cncf email address, I am not sure whether the vulnerability information is submitted to the foundation or the project maintainer, but the tidb security team may not be able to obtain the vulnerability information in the first time.

Reply:
So the tidb security team cares more about `github.com/tikv/pd` but less about the security issues of `github.com/tikv/tikv`?

Reply:
To clarify: I didn't suggest using tidb or CNCF email addresses, I just think tikv/pd should be consistent with tikv/tikv because they both belong to the tikv organization.

Reply:
There are two reasons for submitting this PR. At first, currently PD does not have a security policy to clearly inform users how to submit vulnerabilities; secondly, PD is a key component of tidb, and I personally think that the tidb security team should respond to its vulnerabilities. Anyway, the final choice of security policy is up to the maintainer to decide.

Reply:
I still don't understand why tikv and pd have to be handled differently.